src/content/docs/Build Your Project/project-config.md

```diff
@@ -59,7 +122,7 @@ List of C3 libraries (".c3l") to use when compiling the target.
 
 #### sources
 
-List of source files to compile.
+List of source files to compile and for tests which will run.
```

Comment:
This is not correct. Test sources can be defined separately. This is not implemented I think.